_posts/2020-3-27-SciML.md

```diff
@@ -0,0 +1,392 @@
+---
+layout: post
+title:  "SciML: An Open Source Software Organization for Scientific Machine Learning"
+date:   2020-3-27 10:00:00
+categories:
+---
+
+Scientific Machine Learning, abbreviated SciML, has been taking the academic
+world by storm as an interesting blend of traditional scientific mechanistic
+modeling (differential equations) with machine learning methodologies like
+deep learning. While traditional deep learning methodologies have had difficulties
+with interpretability and enforcing physical constraints, this blend with
+numerical analysis and differential equations has evolved into a field of research
+with new methods, architectures, and algorithms which overcome these traditional
+problems. Because of this, new software will be required in order to accommodate
+the connection between scientific simulators and scientific machine learning
+training loops.

```

Comment:
Perhaps it's worth mentioning explicitly in the paragraph above that traditional deep learning approaches usually don't encode prior knowledge that the scientist might have about their system?